src/include/planner/delete_plan.h

```diff
@@ -64,7 +65,9 @@ class DeletePlan : public AbstractPlan {
   }
 
  private:
-  void BuildInitialDeletePlan(parser::DeleteStatement *delete_statemenet);
+  void BuildInitialDeletePlan(
+      parser::DeleteStatement *delete_statemenet,
+      concurrency::Transaction *consistentTxn = nullptr);
```

Comment:
It shouldn't be null in psql part. But for JDBC part, it needs it to be null. I didn't change the logic of JDBC part.